test/src/test/java/org/corfudb/integration/cluster/Harness/Node.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.integration.cluster.Harness;
+
+import lombok.Getter;
+
+/**
+ * An abstraction for a corfu server/node.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+public class Node {
+
+    @Getter
+    final String address;
+
+    @Getter
+    final String clusterAddress;
+
+    @Getter
+    final String logPath;
+
+    final public Action shutdown;
+
+    final public Action start;
+
+    public Node(String address, String clusterAddress, String logPath) {
+        this.address = address;
+        this.clusterAddress = clusterAddress;
+        this.logPath = logPath;
+        this.shutdown = new ShutdownAction(this);
```

Comment:
Could we automatically discover all "Actions" and register all to a node, without the need of doing it manually? Use reflection.. new actions will be added and they all are assumed to be applied on nodes.